recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -2,9 +2,9 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
 
 CONNAGTIVE_ROOT_AUTHENTICATION ??= "password"
 
-SRC_URI_append = " file://sshd_check_keys_connagtive"
+SRC_URI_append_protectionshield = " file://sshd_check_keys_connagtive"
```

Comment:
I do not think that this will work, as I cannot find `protectionshield` in `OVERRIDES`, neither in meta-phytec, meta-yogurt nor in meta-connagtive. `OVERRIDES` only contains the protectionshield levels itself, like `shieldlow`.